### PR TITLE
PR title: Homework5: Vladislav Savchuk

### PR DESCRIPTION
**Pull request by VladislavSavchuk**
## Added a directory with the tasks solutions files:

- Replace the '#' character with the '/' character in the string 'www.my_site.com#about'

- Write a program that adds 'ing' to words

- In the string 'Ivanou Ivan', swap the words => 'Ivan Ivanou'.

- Write a program that removes a space at the beginning, at the end of a line

- Personal names always start with an uppercase letter followed by lowercase letters. Correct the given proper name to match this statement. “pARiS” >> “Paris”

### CODE REVIEW

File: homework5/adds_ing_to_words.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        if word[-1] == 'e':
+            return word[:-1] + 'ing'
+        # В остальных случаях просто добавляем 'ing' в конец слова
+        return word + 'ing'
+
+    return word
+
+
+# Создаем строку со словами
+MY_STRING = "study spin prefer die travel see"
```
Comment (Stml89) [inline homework5/adds_ing_to_words.py]: That is not a constant https://realpython.com/python-constants/#why-use-constants

File: homework5/name_correction.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+"""
+
+
+def name_correction():
+    """This function corrects a string starting with an uppercase letter"""
+    # создаем строку в разных регистрах
+    capital = "pARiS"
+
+    # проводим проверку, что строка состоит только из букв
+    if capital.isalpha():
```
Comment (Stml89) [inline homework5/name_correction.py]: Why do you need to check it? If there is a number, it won't be converted?

**Comment by Stml89** [comment]
Reply to comment and you can merge it

**Comment by VladislavSavchuk** [comment]
fixed errors and corrected the code in the files

**Pull request merged**